hc/accounts/tests/test_login.py

```diff
@@ -13,20 +13,22 @@ def test_it_sends_link(self):
         session = self.client.session
         session["welcome_code"] = str(check.code)
         session.save()
-
+        self.assertEqual(Profile.objects.count(), 0)
+        # count before
         form = {"email": "alice@example.org"}
 
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
 
         ### Assert that a user was created
-
+        self.assertEqual(Profile.objects.count(), 1)
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject, 'Log in to healthchecks.io')
         ### Assert contents of the email body
-
+        assert "To log into healthchecks.io" in mail.outbox[0].body
```

Comment:
We should take advantage of assertion methods available in the django TestCase class. https://docs.djangoproject.com/en/1.10/topics/testing/tools/#django.test.SimpleTestCase.assertContains